tests/unit/routers/test_platform.py

```diff
@@ -35,7 +39,7 @@ def test_read_root(anon_client):
     assert len(caplogs) == 2
 
 
-def test_read_version(anon_client):
+def test_read_version(anon_client, set_app_root_dir):
```

Comment:
Is this the only test where `set_app_root_dir` is used? If so, should we handle the setup / teardown in the test directly?